sdks/python/apache_beam/io/gcp/bigquery_file_loads.py

```diff
@@ -826,7 +836,8 @@ def _load_data(
             TriggerCopyJobs(
                 create_disposition=self.create_disposition,
                 write_disposition=self.write_disposition,
-                test_client=self.test_client),
+                test_client=self.test_client,
+                step_name=step_name),
```

Comment:
Do we know the format of the step_name. Is this just what the SDK harness is referring to the step as?

IIRC these did not reflect something meaningful, in the original graph to the user. So we probably don't want to show them

I know the pcollections in python at least were very opaque "pcollection1", "pcollection2", etc.

Normally we rely on the DF RunnerHarness to translate those before sending to the UI (ex: on metrics)

Reply:
in this case I believe it's pretty useful. The step would be `UserTransform/ReadFromBigQuery/_CustomBigQuerySource` or whatever.